### ISSUE TITLE
Answers to confirm work?

### ISSUE THREAD

**Issue by dbolesta** [microsoft/frontend-bootcamp#126]
Am I missing something, or are there no answers anywhere to see if what you've done is correct / help find a solution when you get stuck?

**Comment by micahgodbolt** [comment]
This content was presented by two of us. We walked the students through the solutions. I agree adding solutions would help with self service.

**Comment by dbolesta** [comment]
Ah okay. I just wasn't sure if I was missing anything between all the READMEs. I think it would add a lot of value to this tutorial, though I understand if it's not feasible in the near-term.

**Comment by 55asshole** [comment]
I'm confused about this and I would appreciate if you inform me of any kind
of Help I

Timothy Sinko

**Comment by micahgodbolt** [comment]
Yeah, I originally had a few of the lessons with "demo start", "student start" and "final code", but it was a bit confusing during the workshop portion.

If you have exercises you aren't sure if you got right, I'd be curious to know which ones and to see if we can make the instructions more clear.

**Comment by mohsinonxrm** [comment]
@micahgodbolt , are there any recordings available for the sessions? I'd like to go through these activities.

**Comment by micahgodbolt** [comment]
not publicly unfortunately

**Comment by mohsinonxrm** [comment]
@micahgodbolt , any way to get those? make payment etc.?

**Comment by micahgodbolt** [comment]
they were recording during a private company training, so unfortunately not.

**Comment by kenotron** [comment]
@micahgodbolt - wonderful idea to ping Jenna about maybe getting some funding to do it for public consumption (maybe part of the MS Learning side of things??)

**Comment by mohsinonxrm** [comment]
@kenotron , @micahgodbolt , that'll certainly drive more adoption and provide feedback for the Office UI framework.
